Remotion/Validation/Core/Validators/ScalePrecisionValidator.cs

```diff
@@ -54,76 +72,27 @@ protected override bool IsValid (PropertyValidatorContext context)
       return false;
     }
 
+    private string IgnoreTrailingZerosIfSet (string input)
+    {
+      if (IgnoreTrailingZeros.HasValue && IgnoreTrailingZeros.Value)
```

Comment:
Hmm...I'm not sure of .net decimal can _have_ trailing zeros

Reply:
You're right.